api/pkg/backend/router.go

```diff
@@ -35,4 +35,18 @@ func RegisterRouter(ws *restful.WebService) {
 		Doc("Get backend types")
 	ws.Route(ws.POST("/{tenantId}/encrypt").To(handler.EncryptData)).
 		Doc("Encrypt the data")
+
+	ws.Route(ws.GET("/{tenantId}/tiers/{id}").To(handler.GetTier)).
+		Doc("Show tier details")
+	ws.Route(ws.GET("/{tenantId}/tiers").To(handler.ListTiers)).
+		Doc("list tiers")
+	ws.Route(ws.POST("/{tenantId}/tiers").To(handler.CreateTier)).
+		Doc("Create tier")
+	ws.Route(ws.PUT("/{tenantId}/tiers/{id}").To(handler.UpdateTier)).
+		Doc("Update tier")
+	ws.Route(ws.DELETE("/{tenantId}/tiers/{id}/{backendId}").To(handler.DeleteTierBackend)).
```

Comment:
why there is new API for deleting. It should be added in UpdateTier only

Reply:
removed